_locales/_untranslated_en.json

```diff
@@ -22,5 +22,8 @@
   },
   "pref_remove_profile_photo": {
     "message": "Remove profile photo"
+  },
+  "copy_json": {
```

Comment:
maybe there could be more context to help the translators, isn't absolutely necessary.